sechub-integrationtest/src/test/java/com/mercedesbenz/sechub/integrationtest/scenario12/PDSWebScanJobScenario12IntTest.java

```diff
@@ -22,7 +22,7 @@
 import com.mercedesbenz.sechub.integrationtest.internal.IntegrationTestFileSupport;
 
 /**
- * Integration test doing code scans by integration test servers (sechub server,
+ * Integration test doing web scans by integration test servers (sechub server,
```

Comment:
Please add a check at the end that 
```
Line: 88  SecHubScanConfiguration returnedConfiguration
```
does contain all given header information inside result as well.

So it is tested that the DAST PDS solution has got this inside its configuration...